packages/server/lib/screenshots.ts

```diff
@@ -278,6 +294,7 @@ const getBuffer = function (details) {
 
   return Promise
   .promisify(details.image.getBuffer)
+  // @ts-ignore
```

Comment:
I don't have a recommendation on how to fix this one - I think it  starts to open a big can of worms re: our lack of type definitions for automation commands, which would get us the correct type for `details`.

In the meantime, an inline promisify might work, like

```
return new Promise((resolve, reject) => {
  details.image.getBuffer(Jimp.AUTO, (err, val) => {
    if (err) {
      reject(err)
    } else {
      resolve(val)
    }
  }
})
```

This could also work, but the explicit any is unfortunate:

```
 return Promise.promisify(
    (details.image.getBuffer as (arg: typeof Jimp.AUTO) => any).bind(details.image),
  )(Jimp.AUTO)
```

Reply:
Another problem here is that Jimp.getBuffer's type as it is defined in Jimp only accepts strings, but Jimp.AUTO is -1. This *is* valid, but I believe the types are wrong.

Reply:
[`7d6e3a0` (#30758)](https://github.com/cypress-io/cypress/pull/30758/commits/7d6e3a0b960a487f596937ecb85371a3e39534eb)